hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneAtRestEncryption.java

```diff
@@ -520,12 +580,21 @@ private void testMultipartUploadWithEncryption(OzoneBucket bucket,
 
   private void testMultipartUploadWithEncryption(OzoneBucket bucket,
       int numParts, boolean isStream) throws Exception {
+    testMultipartUploadWithEncryption(bucket, numParts, false, false);
+  }
```

Comment:
Callers of this method in other parts of the test are passing `isStream` as true and false, but their inputs are now always being converted to `false`, so there's no testing of MPU with streaming.